docs/contrib/contributing.rst

```diff
@@ -52,7 +52,7 @@ This the general workflow for contributing to Scala Native.
 
 5.  After the review, you should resolve issues brought up by the reviewers as
     needed (amending or adding commits to address reviewers' comments),
-    iterating until the reviewers give their thumbs up, the "LGTM" (acronym for
+    iterating until the reviewers give their thumbs up, the "LONGTIME" (acronym for
```

Comment:
Do we use `LGTM` like the original or `LONGTIME`? https://google.github.io/eng-practices/review/reviewer/speed.html

Reply:
Good catch!  I have no idea where "LONGTIME" came from!  Will fix on my
next activity cycle (late tomorrow, Friday).

At least now I know that somebody is reading this stuff ;-)

Reply:
Fixed now.  Sorry for the delay. Fell off the bottom of the TODO list.